src/app/components/about/about.component.scss

```diff
@@ -51,20 +52,41 @@
     padding: 0 36px;
     height: 605px;
     box-sizing: border-box;
-    .banner-title,
-    p {
-      font-size: rem(14px) !important;
-      text-align: center;
+
+    .about-warp .about-row-logo {
+      margin: auto;
+    }
+
+    .about-warp .stores {
```

Comment:
When nesting in SCSS, use the & placeholder before each selector, like so:
& .about-warp {
    & .stores{
  ..........    
   }
}